### ISSUE TITLE
Dimmer slider is not working

### ISSUE THREAD

**Issue by Pheerink** [robertklep/name.klep.sonoff#23]
I have a PS 16 DZ dimmer flashed with Tasmato. Everything works fine with the Tasmato software. I can dim and toggle no problems! https://github.com/arendst/Sonoff-Tasmota/wiki/PS-16-DZ-Dimmer. Then I tried to get this in Homey and have installed the dimmer with the Sonoff generic Tasmato option.  It is  installed and  I'm able to switch  the dimmer On and Off via Homey. Homey is also providing me a slider to change the intensity but when I use it, the dimmer doesn't react. Any ideas?

**Comment by robertklep** [comment]
I don't have any dimmer devices myself, so support for dimming is not implemented (even though the dimming capability is added).

**Comment by Pheerink** [comment]
That's a pity. For what dimmer did you implement the dimming if you say it has the dimming capability?

**Comment by robertklep** [comment]
No specific device. The app can see that a device supports dimming, and adds this capability to the Homey device that it creates, but it doesn't implement the actual dimming. To be honest, it probably shouldn't be adding the `dim` capability to the Homey device until it has actually implemented support for it.

**Comment by Pheerink** [comment]
I agree with your last statement, however Is dimming on your list to implement or are you only focussing on the Sonoff line. Thanks for all your help it is highly appreciated.

**Comment by robertklep** [comment]
Ideally, I want a totally generic Tasmota driver that's able to find out which capabilities a Tasmota device has, and map all of those capabilities to a Homey devices.

However, my Homey got bricked a few weeks ago and I moved all of my home automation to Home Assistant, so I'm not actively working on Homey apps at the moment (my Homey got replaced by Athom, but I'm quite happy with Home Assistant so haven't yet had a reason to go back to Homey). That may change in the future, but no guarantees.

**Comment by Pheerink** [comment]
I have "played" a couple of years with Domoticz but the problem is that you need a lot of add-ons to get everything working. Homey is providing you these add-ons in one device. If you limit your home devices to wifi it's fine but I have RF 433 - zigbee Z-wave, infrared etc.

**Comment by robertklep** [comment]
I have 433Mhz (some, phasing it out), Zigbee and Z-Wave. All works great on HA 👍🏻

To get your dimmer working fast, you might be able to use the MQTT Client app on Homey combined with a virtual device (flow trigger for "dimmer" messages on the correct topic to set the virtual device's `dim` capability, and flow trigger on the `dim` capability to send a "dimmer" message to your device).

**Comment by Baldhor** [comment]
I played around and did this : https://github.com/Baldhor/tasmota
I control my 5 shutter with it (Teepao device), and it should be compatible with any shutter device running tasmota firmware.

Hope you don't mind @robertklep :)

**Comment by robertklep** [comment]
I don't mind at all! (although my name is "Klep" and not "Kelp" 😜)
